oneview_redfish_toolkit/tests/blueprints/test_computer_system.py

```diff
@@ -247,12 +254,17 @@ def test_get_computer_system_unexpected_error(
     def test_get_computer_system_server_profile(self, g):
         """Tests ComputerSystem with a known Server Profile"""
 
+        server_profile = copy.deepcopy(self.server_profile)
+        server_profile["localStorage"]["sasLogicalJBODs"].pop(0)
+
         # Create mock response
-        g.oneview_client.server_profiles.get.return_value = self.server_profile
+        g.oneview_client.server_profiles.get.return_value = server_profile
         g.oneview_client.server_hardware.get.return_value = \
             self.server_hardware
         g.oneview_client.server_hardware_types.get.return_value = \
             self.server_hardware_types
+        g.oneview_client.sas_logical_jbods.get_drives.return_value = \
```

Comment:
We should include the assert to that call too.
Like: 
`g.oneview_client.sas_logical_jbods.get_drives.assert_called_with(some_value_here)`

And the same to the other tests that are applying a mock this call.